src/main/java/competition/subsystems/vision/VisionSubsystem.java

```diff
@@ -162,7 +164,7 @@ public Optional<EstimatedRobotPose> getRearPhotonVisionEstimatedPose(Pose2d prev
         }
     }
 
-    public boolean isEstimatedPoseReliable(EstimatedRobotPose estimatedPose) {
+    public boolean isEstimatedPoseReliable(EstimatedRobotPose estimatedPose, Pose2d previousEstimatedPose) {
```

Comment:
there's a unit test on this method you'll need to update for this change (that's why the build is failing):
> D:\a\1\s\src\test\java\competition\subsystems\vision\VisionSubsystemTest.java:22: error: method isEstimatedPoseReliable in class VisionSubsystem cannot be applied to given types;
        assertFalse(subsystem.isEstimatedPoseReliable(new EstimatedRobotPose(